src/PokemonGymImpl.java

```diff
@@ -205,8 +208,8 @@ public void gymOwnerAttacks(Pokemon gymPokemon, Pokemon pokemon){
                 String attack = fire.getAttacks().get(randomAttackByGymOwner());
                 switch (attack) {
                     case "inferno" -> fire.inferno(gymPokemon, pokemon);
-                    case "pyroBall" -> fire.pyroBall(gymPokemon, pokemon);
-                    case "fireLash" -> fire.fireLash(gymPokemon, pokemon);
+                    case "pyroball" -> fire.pyroBall(gymPokemon, pokemon);
```

Comment:
Goede fix. De exacte schrijfwijze is belangrijk in deze opdracht, hoewel je het misschien ook "gewoon"  in je pokemon subklassen anders had kunnen schrijven